schema/BUSINESS_LABOR_NON_CONTRIBUTION_ACCOUNT.json

```diff
@@ -6,6 +6,7 @@
     "description": "Non-contribution Account (17)",
     "type": "object",
     "required": [
+        "report_type",
```

Comment:
This is going to cause problems with the current app.  Transactions don't have a report_type property.  I think we'll need to figure out how to factor in the current report that we're validating into the transaction validation.  We may need to update the data JUST before we pass it into `FecSchemaValidatorSerializerMixin.validate()` and `SchemaUtils.jsonSchemaValidator()` to specify which report we're validating against

Reply:
Ok. Made those three changes.
Removed report_type from required
Changed report_type from const to string
Added max_length of 5